test/language/expressions/assignmenttargettype/parenthesized-primaryexpression-objectliteral.js

```diff
@@ -1,22 +0,0 @@
-// This file was procedurally generated from the following sources:
-// - src/assignment-target-type/primaryexpression-objectliteral.case
-// - src/assignment-target-type/invalid/parenthesized.template
-/*---
-description: PrimaryExpression ObjectLiteral; Return invalid. (ParenthesizedExpression)
-esid: sec-grouping-operator-static-semantics-assignmenttargettype
-flags: [generated]
-negative:
-  phase: parse
-  type: SyntaxError
-info: |
-    ParenthesizedExpression: (Expression)
-
-    Return AssignmentTargetType of Expression.
----*/
-
-$DONOTEVALUATE();
-
-function _() {
-  ({}) = 1;
```

Comment:
this test is valid for the same reason of the test above, also deleted in this PR, going through the same route described in the newly introduced test.